src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -533,6 +533,24 @@ impl AstDisplay for CreateSchemaStatement {
 }
 impl_display!(CreateSchemaStatement);
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct ConnectionDefaultAwsPrivatelink<T: AstInfo> {
+    pub connection: T::ItemName,
+    pub port: Option<u16>,
+}
+
+impl<T: AstInfo> AstDisplay for ConnectionDefaultAwsPrivatelink<T> {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        f.write_node(&self.connection);
+        if let Some(port) = self.port {
+            f.write_str(" ( PORT ");
+            f.write_node(&display::escape_single_quote_string(&port.to_string()));
+            f.write_str(" )");
```

Comment:
No need for the extra whitespace

```suggestion
            f.write_str(" (PORT ");
            f.write_node(&display::escape_single_quote_string(&port.to_string()));
            f.write_str(")");
```